contracts/hydro/src/state.rs

```diff
@@ -17,8 +64,8 @@ pub struct Constants {
     pub hub_transfer_channel_id: String,
     pub icq_update_period: u64,
     pub paused: bool,
-    pub is_in_pilot_mode: bool,
```

Comment:
Could this be used in our front end and cause issues?

Reply:
Doesn't appear in the frontend code as far as I can see.